src/SFA.DAS.LevyTransferMatching.Api.UnitTests/Controllers/PledgeTests/GetPledgeApplicationsTests.cs

```diff
@@ -45,8 +45,8 @@ public async Task GetApplications_Returns_GetApplicationsResponse()
             Assert.IsNotNull(okObjectResult);
             Assert.IsNotNull(response);
             Assert.AreEqual(_queryResult.Applications.Count(), response.Applications.Count());
-            Assert.IsTrue(response.PledgeRemainingAmount >= 0);
-            Assert.IsTrue(response.PledgeTotalAmount >= 0);
+            Assert.GreaterOrEqual(response.PledgeRemainingAmount, _queryResult.RemainingAmount);
+            Assert.GreaterOrEqual(response.PledgeTotalAmount, _queryResult.TotalAmount);
```

Comment:
I don't quite follow - the values in _queryResult are valid if they are _greater_ than those in the response?

Reply:
I think equal is the best test here.  Changes done.  Ready for review again.